### PR TITLE
qa_crowbarsetup: testing cinder setting rbd_flatten_volume_from_snaps

### PR DESCRIPTION
**Pull request by sjamgade**
Needs to be tested together https://github.com/crowbar/crowbar-openstack/pull/883

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -2926,6 +2926,10 @@ function custom_configuration
                 proposal_set_value cinder default "['attributes']['cinder']['enable_v2_api']" "true"
             fi
 
+            if [[ $want_cinder_rbd_flatten_snaps ]] ; then
+                proposal_set_value cinder default "['attributes']['cinder']['volumes'][0]['${cinder_backend}']['flatten_volume_from_snaps']" "true"
```
Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: this value can't only be true if the backend is ceph, isn't?

Comment (sjamgade) [inline scripts/qa_crowbarsetup.sh]: Done !

**Comment by sjamgade** [comment]
flattensnaps12  was the correct test-build, which failed for other tests.

**Comment by sjamgade** [comment]
Do not merge until https://github.com/crowbar/crowbar-openstack/pull/883 is merged

**Comment by stefannica** [comment]
Re-tested this after crowbar/crowbar-openstack#883 has been merged. It fixes the ec2api.tests.functional.api.test_snapshots.SnapshotTest tempest test cases.

**Comment by aplanas** [comment]
Added the test here: https://github.com/SUSE-Cloud/automation/pull/2020

**Pull request merged**